code generators/de.fhdo.lemma.model_processing.code_generation.container_base/src/main/java/de/fhdo/lemma/model_processing/code_generation/container_base/file/docker/Service.xtend

```diff
@@ -0,0 +1,48 @@
+package de.fhdo.lemma.model_processing.code_generation.container_base.file.docker
+
+import java.util.Map
+import java.util.List
+import org.eclipse.xtend.lib.annotations.Accessors
+import java.util.LinkedList
+
+/**
+ * This POJO equivalent represents the data structure of a service in a docker-compose
+ * file in yaml format. It is used by the snakeyaml framework for parsing an existing docker
+ * compose file.
+ *
+ * @author [Philip Wizenty](mailto:philip.wizenty@fh-dortmund.de)
+ */
+class Service {
+    @Accessors
+    String build
+    @Accessors
+    String image;
+    @Accessors
+    String restart;
+    @Accessors
+    String container_name;
```

Comment:
camelCase

Reply:
This attribute relates to a docker-compose service type and needs to be written in this way to function properly in combination with the snakeyaml framework.

Reply:
Then please document that here.